src/leiningen/new/cryogen.clj

```diff
@@ -5,66 +5,116 @@
+                       ["resources/templates/themes/blue/html/view.html" (render "themes/blue/html/view.html")]
+                       ["resources/templates/themes/blue/html/sheet.html" (render "themes/blue/html/sheet.html")]
+                       ;css
+                       ["resources/templates/themes/blue/css/gorilla-repl/output.css" (render "themes/blue/css/gorilla-repl/output.css")]
+                       ["resources/templates/themes/blue/css/gorilla-repl/viewer.css" (render "themes/blue/css/gorilla-repl/viewer.css")]
+                       ["resources/templates/themes/blue/css/gorilla-repl/worksheet.css" (render "themes/blue/css/gorilla-repl/worksheet.css")]
+                       ;js needed by gorilla-repl
+                       ["resources/templates/themes/blue/js/gorilla-repl/jslib/d3/d3.v3.min.js" (render "themes/blue/js/gorilla-repl/jslib/d3/d3.v3.min.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/jslib/d3/d3.geo.projection.min.js" (render "themes/blue/js/gorilla-repl/jslib/d3/d3.geo.projection.min.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/jslib/uuid/uuid.core.js" (render "themes/blue/js/gorilla-repl/jslib/uuid/uuid.core.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/jslib/vega/vega.1.3.3.min.js" (render "themes/blue/js/gorilla-repl/jslib/vega/vega.1.3.3.min.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js/worksheetParser.js" (render "themes/blue/js/gorilla-repl/js/worksheetParser.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js/utils.js" (render "themes/blue/js/gorilla-repl/js/utils.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js/mathJaxViewer.js" (render "themes/blue/js/gorilla-repl/js/mathJaxViewer.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js/outputViewer.js" (render "themes/blue/js/gorilla-repl/js/outputViewer.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js/renderer.js" (render "themes/blue/js/gorilla-repl/js/renderer.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js-viewer/segment-viewer.js" (render "themes/blue/js/gorilla-repl/js-viewer/segment-viewer.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js-viewer/worksheet-viewer.js" (render "themes/blue/js/gorilla-repl/js-viewer/worksheet-viewer.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js-viewer/github.js" (render "themes/blue/js/gorilla-repl/js-viewer/github.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js-viewer/bitbucket.js" (render "themes/blue/js/gorilla-repl/js-viewer/bitbucket.js")]
+                       ["resources/templates/themes/blue/js/gorilla-repl/js-viewer/main-viewer.js" (render "themes/blue/js/gorilla-repl/js-viewer/main-viewer.js")]
```

Comment:
I would suggest moving these resources out of the `blue` theme directory and into a directory under the `templates` directory instead if possible. 

For example, `["resources/templates/js/gorilla-repl/js-viewer/main-viewer.js" (render "js/gorilla-repl/js-viewer/main-viewer.js")]`